index.js

```diff
@@ -1,7 +1,14 @@
 const TestRunner = require('./src/TestRunner')
+const fs = require('fs')
 
 exports.runTests = async function (options) {
   return new Promise((resolve, reject) => {
+    const configPath = '.mocha-puppeteer-config.js'
+    if (fs.existsSync(configPath)) {
+      const { mochaOptions, puppeteerOptions } = require(`./${configPath}`)
```

Comment:
Alternatively, you can just `require` the `configPath` and wrap it with a `try/catch`. If we catch an error, that means that no config was found and we can just carry on as usual. 

Also, relative paths won't work in this situation, you will need to use `process.cwd()` to grab the location where the command is being run.